app/src/main/java/com/android/sample/ui/map/Map.kt

```diff
@@ -131,4 +155,119 @@ fun MapScreen(
             tabList = LIST_TOP_LEVEL_DESTINATION,
             selectedItem = Route.MAP)
       })
+
+  if (showBottomSheet) {
+    ModalBottomSheet(
+        onDismissRequest = { showBottomSheet = false },
+    ) {
+      Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
+        // Ligne pour la croix en haut à droite
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(8.dp),
```

Comment:
Again, take a look at C file
```suggestion
            modifier = Modifier.fillMaxWidth().padding(STANDARD_PADDING.dp),
```